### PR TITLE
tests: add aliases integration test.

### PR DESCRIPTION
**Pull request by josepht**
LP:#1646905

### CODE REVIEW

**Comment by sergiusens** [comment]
Oh schnap, I merge @kyrofa's branch #1021

**Comment by kyrofa** [comment]
Yikes, I missed this PR, sorry guys. However, they aren't mutually exclusive-- this didn't fix the executable issue, and that didn't test the YAML (which is probably a good idea). Shall we rebase this and add it?

**Comment by sergiusens** [comment]
@kyrofa and determine if you want `test_alias` or just `test_prime`

File: integration_tests/snaps/alias/hello.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (kyrofa) [inline integration_tests/snaps/alias/hello.sh]: This is already in `bin/`, so you can remove it here.

File: integration_tests/test_snap.py [hunk 2]
```diff
@@ -169,3 +170,20 @@ def test_error_on_bad_yaml(self):
             "Issues while validating snapcraft.yaml: found character '\\t' "
             "that cannot start any token on line 13 of snapcraft.yaml",
             str(error.output))
+
+    def test_aliases(self):
```
Comment (kyrofa) [inline integration_tests/test_snap.py]: Can you instead put this test in `test_alias.py`? You can probably merge it with the test that's already there.

Comment (josepht) [inline integration_tests/test_snap.py]: Done.

File: integration_tests/snaps/alias/hello.sh (hunk 1 above)
Comment (josepht) [inline integration_tests/snaps/alias/hello.sh]: Done.

**Pull request merged**